client/src/entrypoints/admin/page-editor.js

```diff
@@ -24,43 +24,13 @@ function initKeyboardShortcuts() {
 
 window.initKeyboardShortcuts = initKeyboardShortcuts;
 
+/**
+ * Ensure any code calling this global function does not throw an error.
+ *
+ * @deprecated RemovedInWagtail60
+ */
+window.updateFooterSaveWarning = () => {};
```

Comment:
> Backwards compatible global `updateFooterSaveWarning`, can add this if needed, but this felt like an internal function and would not be called by custom code.

I don't think it's necessary, this stub function is more than enough 👍

Reply:
Great, I will remove this.